app/views/assays/show.html.erb

```diff
@@ -1,121 +1 @@
-          <b><%= assay_type_text -%>:</b>
-          <%= link_to_assay_type(@assay) -%>
-        </p>
-        <% unless @assay.is_modelling? -%>
-            <p id="technology_type">
-              <b>Technology type: </b>
-              <%= link_to_technology_type(@assay) -%>
-            </p>
-        <% end -%>
-
-        <% if Seek::Config.organisms_enabled %>
-            <%= list_assay_organisms("Organisms", @assay.assay_organisms,{:id=>"organism"}) %>
-        <% end %>        
-
-        <% if Seek::Config.human_diseases_enabled %>
-            <%= list_assay_human_diseases("Human Diseases", @assay.assay_human_diseases,{:id=>"human_disease"}) %>
-        <% end %>
-      </div>
-    </div>
-    <% if @assay.external_asset %>
-      <div class="row">
-        <div class="col-md-12">
-          <%= external_asset_details(@assay) %>
-        </div>
-      </div>
-    <% end %>
-
-    <%= render partial: 'custom_metadata/custom_metadata_attribute_values', locals: {resource: @assay} %>
-
-    <% if ((@assay.is_modelling?) && !@assay.models.empty? && !@assay.data_files.empty?) %><%#MODELLING ASSAY %>
-        <div class="data_model_relationship">
-          <% unless @assay.models.empty? %>
-              <b><%= @assay.models.size > 1 ? t('model').pluralize : t('model') %>:</b>
-              <br/>
-              <ul class="related_models">
-                <% @assay.models.each do |m| %>
-                    <li>
-                      <% if m.can_view? %>
-                          <%= link_to truncate(h(m.title), :length=>110), show_resource_path(m) %>
-                      <% else %>
-                          <span class="none_text">Hidden <%= t('model') %></span>
-                      <% end %>
-                    </li>
-                <% end %>
-              </ul>
-          <% end %>
-          <% unless @assay.data_files.empty? %>
-              <b>Data:</b>
-              <br/>
-              <ul class="related_data_files">
-                <% @assay.data_files.each do |d| %>
-                    <li>
-                      <% if d.can_view? %>
-                          <%= link_to truncate(h(d.title), :length=>110), show_resource_path(d) %>
-                      <% else %>
-                          <span class="none_text">Hidden <%= t('data_file') %></span><%= d.relationship_type(@assay) ? (" <span class='relationship_type'>(" + d.relationship_type(@assay).title + ")</span>").html_safe : "" %>
-                      <% end %>
-                    </li>
-                <% end %>
-              </ul>
-          <% end %>
-        </div>
-    <% end %>
-    <%= render partial: 'general/isa_graph', locals: { root_item: @assay, options: { depth: 4 } } %>
-  </div>
-
-  <div class="col-md-3 col-sm-4">
-    <%= render :partial=>"assets/resource_main_content_right",:locals => { :resource => @assay } %>
-  </div>
-</div>
-
-
-<%= render :partial=>"general/items_related_to",:object => @assay %>
+<%= render :partial=>"show" -%>
```

Comment:
I don't really like this way. I think it would be better if you created a special minimal layout (https://github.com/seek4science/seek/blob/f8b05ca07bbce8b8db2c4370d7ce288ee515c7de/app/views/layouts/application.html.erb) and just rendered the view in an iframe or something.